specification/domainservices/resource-manager/Microsoft.AAD/stable/2021-05-01/domainservices.json

```diff
@@ -0,0 +1,1113 @@
+            "name": "TlsV1",
+            "modelAsString": true
+          }
+        },
+        "syncNtlmPasswords": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not SyncNtlmPasswords is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "SyncNtlmPasswords",
+            "modelAsString": true
+          }
+        },
+        "syncKerberosPasswords": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not SyncKerberosPasswords is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "SyncKerberosPasswords",
+            "modelAsString": true
+          }
+        },
+        "syncOnPremPasswords": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not SyncOnPremPasswords is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "SyncOnPremPasswords",
+            "modelAsString": true
+          }
+        },
+        "kerberosRc4Encryption": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not KerberosRc4Encryption is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "KerberosRc4Encryption",
+            "modelAsString": true
+          }
+        },
+        "kerberosArmoring": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Disabled",
+          "description": "A flag to determine whether or not KerberosArmoring is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "KerberosArmoring",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "ConfigDiagnostics": {
+      "description": "Configuration Diagnostics",
+      "properties": {
+        "lastExecuted": {
+          "type": "string",
+          "format": "date-time-rfc1123",
+          "description": "Last domain configuration diagnostics DateTime"
+        },
+        "validatorResults": {
+          "type": "array",
+          "description": "List of Configuration Diagnostics validator results.",
+          "items": {
+            "$ref": "#/definitions/ConfigDiagnosticsValidatorResult"
+          }
+        }
+      }
+    },
+    "ConfigDiagnosticsValidatorResult": {
+      "description": "Config Diagnostics validator result data",
+      "properties": {
+        "validatorId": {
+          "type": "string",
+          "description": "Validator identifier"
```

Comment:
>Validator identifier [](http://example.com/codeflow?start=26&length=20)

Suggest describing the format (e.g. guid, name, Url) so customer can find it in the docs. Not required for ARM signoff, just a suggestion.